Makefile

```diff
@@ -141,6 +158,21 @@ test: stacker $(REGCLIENT) $(SKOPEO) $(ZOT)
 		$(shell [ -z $(PRIVILEGE_LEVEL) ] || echo --privilege-level=$(PRIVILEGE_LEVEL)) \
 		$(patsubst %,test/%.bats,$(TEST))
 
+.PHONY: check-cov
+check-cov: lint test-cov
+
+.PHONY: test-cov
+test-cov: stacker-cov $(REGCLIENT) $(SKOPEO) $(ZOT)
```

Comment:
is there ever a reason to call the unit tests without coverage?

Reply:
We will make that call once we are able to get coverage data.

Reply:
We definitely want "go-test" (it does its own -cover etc) and bats tests separate.

Reply:
I would not optimize prematurely here. Let's see how this is working for us first.